package.json

```diff
@@ -55,8 +55,7 @@
     "stylelint-scss": "^3.18.0"
   },
   "devDependencies": {
-    "eslint-config-skyscanner": "^8.0.0",
-    "format-package": "^5.2.0",
+    "format-package": "^6.1.0",
```

Comment:
Upgrading `format-package` as this version also uses Prettier 2, this way there's a single version installed